pages/1.11/release-notes/1.11.6/index.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.6
+title: Release Notes for 1.11.6
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.11.6
+---
+
+DC/OS 1.11.6 was released on September 25, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.6/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+# Notable Changes in DC/OS 1.11.6
+
+DC/OS 1.11.6 includes the following components:
+- Apache Mesos 1.5.x [change log](https://github.com/apache/mesos/blob/19d17cec3e797758e76c081efb68867d440ed4d3/CHANGELOG).
+- Marathon 1.6.544 [change log](https://github.com/dcos/dcos/pull/3289).
+- Metronome 0.4.3 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.3).
+
+- DCOS-22310/DCOS-40495/DCOS-41282 - Update DC/OS UI for [1.11+v1.21.0](https://github.com/dcos/dcos-ui/blob/1.11+v1.21.0/CHANGELOG.md).
+- DCOS_OSS-4106/DCOS_OSS-4109 - Update to [ZooKeeper 3.4.13](https://zookeeper.apache.org/doc/r3.4.13/releasenotes.html). [enterprise type="inline" size="small" /]
+
+
+# Issues Fixed in DC/OS 1.11.6
+
+## GUI
+- DCOS-40779 - Fix movement of form control icons in the search bar. 
+
+## Marathon
+- DCOS-41198 - Replace `poll_marathon_for_app_deployment()` method with `wait_for_deployments_complete()` method in dcos-test-utils.
+
+## Mesos
+- COPS-3527/DCOS_OSS-3921 - Check cache when creating Mesos resources using `make_disk_resources.py`.
+- COPS-3574/DCOS-19598/DCOS-40317/DCOS-41375 - Bump Mesos to nightly [1.5.x dd68c0b](https://github.com/mesosphere/mesos/blob/dd68c0b/CHANGELOG).
+- DCOS-24595/DCOS_OSS-3991 - Add Mesos patches for per-framework metrics.
+- DCOS-40162 - Fix container launch failures that occurred due to Mesos-bridge running out of IPs.
+
```

Comment:
Missing https://jira.mesosphere.com/browse/COPS-3371

Reply:
Added missing tickets and included DCOS tickets for related COPS tickets.